### PR TITLE
Review current version of documentation and code

### PR DESCRIPTION
**Pull request by haidee09**
Any comments about the NGSI Javascript library are welcome: features, documentation, doubts about its implementation (in code), contributions, bug reports and proposals.

### CODE REVIEW

File: doc/ocb/docs/ImplementationInMobileApps.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#Implementation of OCB-sender module in Mobile Apps
```
Comment (taliaga) [inline doc/ocb/docs/ImplementationInMobileApps.md]: missing space

File: doc/adminManual.md [hunk 2]
```diff
@@ -1,35 +1,42 @@
 # Pre-requirements of Installation
 
-In order to use both npm modules of NGSI JS Library its necessary cover the following requirements.
+The pre-requirements for installing and using the modules of NGSI JavaScript Library are the following:
 
 ##Node.js and npm
```
[the remark this answers is not in the document]
Comment (cenidetiot) [reply doc/adminManual.md]: Ok Tomas, resolved. I will check that this does not happen in other documents, sorry

File: doc/ocb/docs/SubscriptionsFunctions.md [hunk 3]
```diff
@@ -13,24 +13,25 @@
...
-cb.getSubscription("5a83c5463fc4dec59e4ef8e2")
+cb.getSubscription("5a83c5463fc4dec59e4ef8e2", headers)
 .then((result) => console.log(result))
 .catch((err) => console.log(err))
 ```
 ## Create Functions.
 
 ### Create Subscription.
-Example
+The `createSubscription()` function is used for register a subscription in the Orion ContextBroker. This function receives as parameter the NGSI subscription. For example:
```
[the remark this answers is not in the document]
Comment (danieltoo) [reply doc/ocb/docs/SubscriptionsFunctions.md]: Yeah, in the [Orion documentation](https://fiware-orion.readthedocs.io/en/master/user/walkthrough_apiv2/index.html#subscriptions) comments that the subscriptions id is returned as a header called "location" something like /v2/subscriptions/57458eb60962ef754e7c0998, I'm already working on that.
Thanks for the suggestion !

File: doc/ngsi/docs/JSONParser.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+* [Parsing an Attribute Value](#parse-a-value)
+* [Parsing an Attribute](#parse-an-attribute)
+* [Parsing an Entity](#parse-an-entity)
+
+### Parsing an Attribute Value
+When you want to parse an attribute value to the format NGSI, you can use the `parseValue()`
+function. This function receives as a parameter the value of the attribute, the data type of this attribute value must be a data type supported by the ngsi-parser module (described in the [Data types supported](../index.md#data-types-supported)).
+Example:
+```js
+	var value = ngsi.parseValue(50)
```
[the remark this answers is not in the document]
Comment (cenidetiot) [reply doc/ngsi/docs/JSONParser.md]: ok tomas, resolved

File: doc/ocb/docs/QueryFunctions.md [hunk 5]
```diff
@@ -3,24 +3,37 @@
...
+})
 .catch((err) => console.log(err))
 ```
-### Query Entities on Area.
-***NOTE***: For this function the use of headers is not available.
-Usage
+### Query Entities on Area
+The `queryEntitiesOnArea()` function is used for retrieving the entities located within a specific area. This function receives the following four parameters:
+
+- **1.-** The coordinates of the polygon of the area (longitude-latitude array)
```
[the remark this answers is not in the document]
Comment (cenidetiot) [reply doc/ocb/docs/QueryFunctions.md]: yes tomas, resolved

File: doc/ngsi/docs/JSONParser.md [hunk 6]
```diff
@@ -0,0 +1,122 @@
+# JSON Parser Functions
+The component JSON parser analyzes the content of complete JSON entities, JSON entity attributes or attributes values, and converts them in objects with FIWARE-NSGIv2 format. The functions of this component are the following:
+
+* [Parsing an Attribute Value](#parse-a-value)
```
[the remark this answers is not in the document]
Comment (cenidetiot) [reply doc/ngsi/docs/JSONParser.md]: no, sorry, the titles were wrong, i've fixed them

File: doc/ngsi/docs/JSONParser.md [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+				"scale":{
+					"value":"Fahrenheit",
+					"type":"Text"
+				}
+			}
+		}
+	}
+
+```
+### Parsing an Entity
```
[the remark this answers is not in the document]
Comment (haidee09) [reply doc/ngsi/docs/JSONParser.md]: We are working on that, thanks for the suggestion!

File: doc/overview.md [hunk 8]
```diff
@@ -1,48 +1,52 @@
 # Introduction
 
-Welcome to the quick start guide of NGSI JavaScript Library! In this brief chapter, you will find general information to help you understand the way that the library is conformed and how it works.
-The NGSI Library manipulates the context entities according to the FIWARE-NGSI v2 API. The specification that is intended to manage the entire lifecycle of context information, including updates, queries, registrations, and subscriptions. 
-The NGSI Library is composed of two npm modules; i.e node packages that can be found in the [official site of npm](https://www.npmjs.com/).
+Welcome to the quick start guide of NGSI JavaScript Library! In this brief chapter, you will find general information to help you understand the way that the library is composed and how it works.
+The NGSI Library manipulates the context entities according to the FIWARE-NGSI v2 API. This specification is intended to manage the entire lifecycle of context information, including updates, queries, registrations, and subscriptions. 
+The NGSI Library is composed of two npm modules: ngsi-parser and ocb-sender, these node packages are in the [official site of npm](https://www.npmjs.com/).
```
[the remark this answers is not in the document]
Comment (haidee09) [reply doc/overview.md]: OK tomas, ready

**Pull request merged**